src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Circle circle = new Circle();
+    private final Square square = new Square();
+    private final IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid();
+    private final RightTriangle rightTriangle = new RightTriangle();
+    private final Rectangle rectangle = new Rectangle();
+
+    public Figure getRandomFigure() {
```

Comment:
Lets create one new Figure in this method

Reply:
I created a new Figure array or that's not what u meant?

Reply:
only one figure should be created. 
for the future, please ask any questions at the Mate platform's chat

Reply:
not fixed